pkg/storage/storagewrappers/caching_test.go

```diff
@@ -54,37 +58,9 @@ func TestReadAuthorizationModel(t *testing.T) {
 	gotModel, err = cachingBackend.ReadAuthorizationModel(ctx, storeID, model.Id)
 	require.NoError(t, err)
 	require.Equal(t, model, gotModel)
-}
-
-func TestSingleFlightFindLatestAuthorizationModelID(t *testing.T) {
```

Comment:
why remove this test? we are still using singleflight 🤔

Reply:
added back